File: 2600. K 件物品的最大和.c
int min(int x, int y){
	if (x < y){
		return x;
	}
	return y;
}

int max(int x, int y){
	if (x > y){
		return x;
	}
	return y;
}


int kItemsWithMaximumSum(int numOnes, int numZeros, int numNegOnes, int k){
	return min(k, numOnes) - max(k-numOnes-numZeros, 0);
}

File: 2600. K 件物品的最大和.go
func max(x, y int) int {
	if x > y {
		return x
	}
	return y
}

func min(x, y int) int {
	if x < y {
		return x
	}
	return y
}

func kItemsWithMaximumSum(numOnes int, numZeros int, numNegOnes int, k int) int {
	return min(k, numOnes) - max(k-numOnes-numZeros, 0)
}




